sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/DeviceNetworkSample.md

```diff
@@ -0,0 +1,15 @@
+# Network retrieve
+
+To get the Network name from a network identifier (IPv4/IPv6/NetworkCode), run the following code.
+
+```C# Snippet:APC_Sample_NetworkRetrievalTest
+string ApcGatewayId = "/subscriptions/abcdefgh/resourceGroups/dev-testing-eastus/providers/Microsoft.programmableconnectivity/gateways/apcg-eastus";
+Uri _endpoint = new Uri("https://your-endpoint-here.com");
+TokenCredential _credential = new DefaultAzureCredential();
```

Comment:
The underscores here are an unusual pattern.  I'd suggest removing them for samples.